### PR TITLE
Update Windows installer

### PR DESCRIPTION
**Pull request by benjaoming**
### Summary

I'd like to track this to go into `develop`. We might not merge it until https://github.com/learningequality/kolibri-installer-windows/pull/93 is in, but let's see.

### Reviewer guidance

n/a

### References

External issues:

https://github.com/learningequality/kolibri-installer-windows/issues/71

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by indirectlylit** [comment]
see previous thread in #4862

File: docker/env.list [hunk 1]
```diff
@@ -5,7 +5,7 @@ EXTRA_REQUIREMENTS
 KOLIBRI_VERSION
 
 # Set the default version of the Windows Installer
-KOLIBRI_WINDOWS_INSTALLER_VERSION=v1.2.1
+KOLIBRI_WINDOWS_INSTALLER_VERSION=download-python.exe
```
Comment (indirectlylit) [inline docker/env.list]: should be pointed at a real version

**Review by indirectlylit — DISMISSED** [review]
@mrpau-julius do you know the state of this PR?

File: docker/env.list (hunk 1 above)
Comment (benjaoming) [reply docker/env.list]: Yup :)

**Comment by benjaoming** [comment]
Tests permitting, this is ready.

**Comment by benjaoming** [comment]
I had managed to sneak in a rebase conflict, rebased a second time.

**Comment by indirectlylit** [comment]
@mrpau-julius or @mrpau-richard would one of you do the final test & review please

**Comment by mrpau-richard** [comment]
I tested the Installer built with this PR using Windows 7 and it's working as expected.

**Review by mrpau-richard — APPROVED** [review]

**Comment by indirectlylit** [comment]
Does this have any user-facing impact, or is it an internal tech update?

**Comment by mrpau-julius** [comment]
The installation of this PR, it works as expected in Windows 10.  @indirectlylit, it is an internal tech update.

**Review by mrpau-julius — APPROVED** [review]

**Comment by indirectlylit** [comment]
great, thanks. Will merge to `develop` then

**Pull request merged**